### PR TITLE
[Docs] Complete translation for src/dev/js/networking.md

### PR DESCRIPTION
**Pull request by pasicopan**
Signed-off-by: pansijia <pasico@163.com>

### CODE REVIEW

**Comment by pasicopan** [comment]
@hydai  please give me some suggestion 😊😊😊

File: docs/book/zh/src/dev/js/networking.md [hunk 1]
```diff
@@ -1 +1,222 @@
 # 网络
+
+QuickJS WasmEdge Runtime 支持 WasmEdge 的[网络 sockets 拓展](https://github.com/second-state/wasmedge_wasi_socket)， 所以JavaScript 程序也可以在网络上建立HTTP 连接。此文将向你展示相关的 [HTTP 客户端](https://github.com/second-state/wasmedge-quickjs/blob/main/example_js/wasi_http_client.js)和 [HTTP 服务端](https://github.com/second-state/wasmedge-quickjs/blob/main/example_js/wasi_http_echo.js)例子.
+
+
+> WasmEdge 的网络 API 是非阻塞的，所以能够开发出强异步 I/O 交互的应用。当网络请求 handler 正在创建一个对外的请求并等待服务应答的时候，应用仍然可以处理另外一个进来的请求。这让单线程应用可以并发处理多个请求。
+
+## JavaScript 客户端网络通讯例子
+
+以下是一个使用 JavaScript 编写的，作为客户端发出网络请求的例子。你可以在 [example_js/wasi_http_client.js](https://github.com/second-state/wasmedge-quickjs/blob/main/example_js/wasi_http_client.js) 中找到源码。以下的代码会向你展示如何发送一个异步 HTTP GET 请求
```
Comment (alabulei1) [inline docs/book/zh/src/dev/js/networking.md:10]: JavaScript 编写的异步客户端的例子？最后一句结束加 。

File: docs/book/zh/src/dev/js/networking.md [hunk 2]
```diff
@@ -1 +1,222 @@
...
+  "url": "http://152.136.235.225/post?a=123"
+}
+```
+
+
+以上应用例子发出了两个 HTTP 请求，一个是 `GET` 请求另一个是 `POST` 请求。该应用会异步等待这两个请求的应答数据，并且哪一个先从服务端返回就先会处理哪个。从日志中你可以看到这两个请求的 handlers 是交错执行的。
+
+## JavaScript 网络服务例子
+
+以下的例子是使用 JavaScript 跑了个在监听了 8000 端口的 TCP 服务器。接受到的网络请求都会被异步处理。你可以在 [example_js/wasi_net_echo.js](https://github.com/second-state/wasmedge-quickjs/blob/main/example_js/wasi_net_echo.js) 中找到源码。
```
Comment (alabulei1) [inline docs/book/zh/src/dev/js/networking.md:91]: 跑 --> 运行？

File: docs/book/zh/src/dev/js/networking.md [hunk 3]
```diff
@@ -1 +1,222 @@
...
+  for (var i = 0; i < 100; i++) {
+    let cs = await s.accept();
+    handle_client(cs);
+  }
+}
+
+server_start();
+```
+
+调用 `server_start()` 方法会在 8000 端口启动一个监听服务。当一个请求进入，会异步传给 `handle_client()` 处理。这意味着当应用返回应答数据后，它又可以处理下一个进来的请求了。
```
Comment (alabulei1) [inline docs/book/zh/src/dev/js/networking.md:123]: function 函数？

File: docs/book/zh/src/dev/js/networking.md [hunk 4]
```diff
@@ -1 +1,222 @@
...
+
+
+使用以下 CLI 命令，就可以在 WasmEdge runtime中运行这段 JavaScript 代码。因为它将作为一个服务运行，你最好是以后台应用的形式启动。
```
Comment (alabulei1) [inline docs/book/zh/src/dev/js/networking.md:126]: runtime 后空一格

File: docs/book/zh/src/dev/js/networking.md [hunk 5]
```diff
@@ -1 +1,222 @@
...
+
+然后你就可以向它发出网络请求，观察运行效果。
+
+```
+$ curl -d "WasmEdge" -X POST http://localhost:8000
+echo:WasmEdge
+```
+
+
+WasmEdge 的 `wasi_net` 包为 JavaScript 应用提供了一种自适应的动态网络栈。在很多高级用法中，我们基于这个包，设计了很多抽象良好的API。在下一章节，我们会带着具体的应用常见，向你展示如何处理HTTP 请求的。在 [React 服务器渲染文章](ssr.md)中，我们还将会讨论一下如何基于这种异步网络的API 来创建一个React 服务器渲染功能。
```
Comment (alabulei1) [inline docs/book/zh/src/dev/js/networking.md:141]: 具体的常见应用？还有就是英文前后空一格

**Review by alabulei1 — COMMENTED** [review]
Thank you for your contribution!

**Comment by pasicopan** [comment]
@alabulei1 please check it from https://github.com/WasmEdge/WasmEdge/pull/1141

**Comment by alabulei1** [comment]
> @alabulei1 please check it from #1141
Done! Thanks. I think we could close this PR and continue working on #1141.

**Pull request closed without merging**